src/js/api/config.js

```diff
@@ -104,25 +105,25 @@ const Config = function(options, persisted) {
     let rateControls = config.playbackRateControls;
 
     if (rateControls) {
-        let rates = [0.5, 1, 1.25, 1.5, 2];
+        let rates = config.playbackRates;
 
         if (_.isArray(rateControls)) {
```

Comment:
Can you update this to use `Array.isArray` in lieu of `_.isArray`? We want to stop using underscore whenever there's a native API that does the same thing.